### ISSUE TITLE
The difference between the versions

### ISSUE THREAD

**Issue by StanislavPRJ** [giltesa/Game-Boy-Advance-SP-USB-C-charging-kit#3]
Good afternoon.
Could you explain the difference between the versions from 1.2 to 1.5?
Unfortunately I do not have the opportunity to order in your store this mod. So have to do it myself.

**Comment by StanislavPRJ** [comment]
Asked and found the answer myself, up to version 1.4.

What then is the difference between 1.5 and the others?

**Comment by giltesa** [comment]
Hello!

I tried to improve the board with each order to the manufacturer (If I see something than can be better):

    2021-04: v1.0
        Initial version.

    2021-05: v1.1
        Changed the USB-C model.
        Changed the resistances for a smaller size.
        Increased the PCB thickness to 1.6mm

    2021-06: v1.2
        Removed the resistances. Internal speaker doesn’t sound with them.
        Decrees the PCB thickness to 1.2mm.

    2021-10: v1.3
        Modified the USB holes to solder it a bit stronger.

    2021-11: v1.4
        Modified the frame to separate each board easier (It doesn’t affect to the board)

    2022-06: v1.4 (June)
        Modified the frame to separate each board even easier (It doesn’t affect to the board)

    2022-12: v1.6
        Added some of the new features from the centered version like the charging compatibility and board shape.

**Comment by StanislavPRJ** [comment]
Thank you. I read the changelog. 
Can you tell me how the posted version 1.5 differs from 1.4? 
As I see it, it has a slightly different scheme? I'm just not really into it, maybe my questions are stupid :)

Or is it enough for me to take the 1.4 version and not complicate things?

**Comment by StanislavPRJ** [comment]
I think I understand that for version 1.5 I need a resistor, right?

**Comment by giltesa** [comment]
You are right, the v1.4 doesn't have the resistor and USB-C to USB-C cables don't work. The v1.5 has it, so these cables/charges are compatible.

**Comment by StanislavPRJ** [comment]
And if I can ask two more questions :)

For version 1.4 I can take the plastic cap model from folder 1.5 or 1.3. Which one will fit?

And is there a freely available model of a plastic plug for the board with a thickness of 2.4 mm?

**Comment by giltesa** [comment]
That depends on the thickness of the board: 

1.2mm:
https://github.com/giltesa/Game-Boy-Advance-SP-USB-C-charging-kit/tree/master/2.%20Renders%2C%203D%20models/Plastic%20Cap/1.%20Half/v15

2.4mm (centered):
https://github.com/giltesa/Game-Boy-Advance-SP-USB-C-charging-kit/tree/master/2.%20Renders%2C%203D%20models/Plastic%20Cap/2.%20Centered/v30%20Half

**Comment by StanislavPRJ** [comment]
Thanks, I figured it out :)
And just now noticed that in the folder with the board version 1.4 no Gerbers, and so should be?
As I understand, I can use GBASP USB-C.brd file to make the board?

**Comment by giltesa** [comment]
I added the gerbers! (they can be modified and genered again with the brd file)

**Comment by StanislavPRJ** [comment]
Thanks for the help!
I will try to install it.

**Comment by StanislavPRJ** [comment]
Tried to have a 2.4 thick board made in China today. 
The manufacturer is swearing that "Failed reason:
The spacing between the castellated holes should be greater than 0.55 mm;".
Is there no way to fix it? Unfortunately it is unlikely that I will be able to order production elsewhere...
![1632387532317t](https://github.com/giltesa/Game-Boy-Advance-SP-USB-C-charging-kit/assets/56292200/94bd3ace-af91-486a-a1a7-eae710701e19)

**Comment by StanislavPRJ** [comment]
Or maybe you can tell me where you made the 2.4 boards?

**Comment by giltesa** [comment]
I currently use the JLCPCB factory. Version 1.4 of the board was produced a year ago, but the factory has since become stricter in enforcing manufacturing rules. While they used to accept orders that didn't comply with the rules a year ago, they no longer do so. The new board versions now include the resistor and also adhere to the manufacturing rules. There is now approximately 0.3048mm more separation between the pads than before:
![imagen](https://github.com/giltesa/Game-Boy-Advance-SP-USB-C-charging-kit/assets/8562315/ea7f7ef8-30a3-47dc-8823-f26f56d0ca96)

**Comment by StanislavPRJ** [comment]
JLCPCB does not allow to make boards thicker than 2.0 :(
So I had to order from PCBWay. 

But there is no headphone support in the 1.5 version, is there?

**Comment by StanislavPRJ** [comment]
Or if I want sound, I don't solder a resistor. 
And if I need Type-C to Type-C, I solder a resistor.
Did I get that right?

**Comment by giltesa** [comment]
All the board versions I have released support charging the battery and using headphones. However, not all of them support USB Type-C to Type-C cables.
The regular version, the one without the centered connector, needs to be manufactured with a thickness of 1.2mm.
The centered version requires a thickness of 2.4mm, and I manufacture it at PCBWay.

I'm not sure what removing the resistors will entail; I haven't tested it.

**Comment by StanislavPRJ** [comment]
I watched your video https://youtu.be/ZznNsDhFl9U

And I understand it uses the 1.5 version with the resistor fitted. Is that correct?

**Comment by StanislavPRJ** [comment]
I watched your video https://youtu.be/ZznNsDhFl9U

And I understand it uses the 1.5 version with the resistor fitted. Is that correct?

**Comment by StanislavPRJ** [comment]
I apologise for my silly questions. I just want to be sure, as manufacturing and delivery will take a long time for me and I don't want to waste it.
Thank you for your replies!

**Comment by giltesa** [comment]
You are right, that video shows the board v1.5 or higher. (I don't remember)

I don't know if the factory will request to add a frame since the board is too small. I never ordered only one board for panel. Something like this:
![imagen](https://github.com/giltesa/Game-Boy-Advance-SP-USB-C-charging-kit/assets/8562315/8724d753-6063-4d09-84ac-29df5b15710c)
